build/config/ozone_extra.gni

```diff
@@ -18,6 +18,12 @@ ozone_external_platform_deps = []
 # ozone_external_platform_test_deps = [ "platform/foo1:foo1_unitests", ... ]
 ozone_external_platform_test_deps = []
 
+if (defined(is_starboard) && is_starboard) {
```

Comment:
@andrewsavage1, are we missing `is_starboard` gn variable defenition?

Reply:
Right now it just isn't defined for chromium builds, which we can potentially change